[0m20:37:14.639015 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107946590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10796a390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10796ad90>]}


============================== 20:37:14.641923 | 60f138c6-e1a2-4e90-80f5-3690ab6f22be ==============================
[0m20:37:14.641923 [info ] [MainThread]: Running with dbt=1.8.0
[0m20:37:14.642233 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/manojgowdavenkatachalamurthy/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m20:37:27.857775 [debug] [MainThread]: Starter project path: /Users/manojgowdavenkatachalamurthy/anaconda3/lib/python3.11/site-packages/dbt/include/starter_project
[0m20:37:27.865595 [info ] [MainThread]: 
Your new dbt project "data_pipeline" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m20:37:27.866083 [info ] [MainThread]: Setting up your profile.
[0m20:38:05.796860 [info ] [MainThread]: Profile data_pipeline written to /Users/manojgowdavenkatachalamurthy/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m20:38:05.812498 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 51.207325, "process_user_time": 3.2891, "process_kernel_time": 0.388545, "process_mem_max_rss": "184975360", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m20:38:05.813248 [debug] [MainThread]: Command `dbt init` succeeded at 20:38:05.813148 after 51.21 seconds
[0m20:38:05.813623 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103502b90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103484a90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103484cd0>]}
[0m20:38:05.813987 [debug] [MainThread]: Flushing usage events
[0m20:42:58.231216 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11870fc50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11874fcd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11874e690>]}


============================== 20:42:58.234327 | 65e715e1-5113-4977-86c4-487cec348cc0 ==============================
[0m20:42:58.234327 [info ] [MainThread]: Running with dbt=1.8.0
[0m20:42:58.234650 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/manojgowdavenkatachalamurthy/.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'invocation_command': 'dbt deps', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m20:42:58.235139 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/manojgowdavenkatachalamurthy/Documents/Data_Analytics/Projects/Data Engineering/Snowflake/ELT/DBT_SF_Airflow/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m20:42:58.236724 [debug] [MainThread]: Resource report: {"command_name": "deps", "command_wall_clock_time": 0.03476121, "process_user_time": 0.592528, "process_kernel_time": 0.125426, "process_mem_max_rss": "107216896", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m20:42:58.237051 [debug] [MainThread]: Command `dbt deps` failed at 20:42:58.237002 after 0.04 seconds
[0m20:42:58.237261 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11870ce10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11875c190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11874f710>]}
[0m20:42:58.237444 [debug] [MainThread]: Flushing usage events
[0m20:43:17.794193 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1055e0290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107848d90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10786ef10>]}


============================== 20:43:17.796266 | d1a247a7-ca92-41fb-8556-6f84905e2319 ==============================
[0m20:43:17.796266 [info ] [MainThread]: Running with dbt=1.8.0
[0m20:43:17.796599 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/Users/manojgowdavenkatachalamurthy/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt deps', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m20:43:17.797108 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/manojgowdavenkatachalamurthy/Documents/Data_Analytics/Projects/Data Engineering/Snowflake/ELT/DBT_SF_Airflow/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m20:43:17.798044 [debug] [MainThread]: Resource report: {"command_name": "deps", "command_wall_clock_time": 0.030307917, "process_user_time": 0.542022, "process_kernel_time": 0.078505, "process_mem_max_rss": "107577344", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m20:43:17.798383 [debug] [MainThread]: Command `dbt deps` failed at 20:43:17.798326 after 0.03 seconds
[0m20:43:17.798623 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10786dbd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10786e910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10786dd90>]}
[0m20:43:17.798820 [debug] [MainThread]: Flushing usage events
